jsocnfig.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
filename typo? haha

Reply:
Yes and actually we don't need this. This was needed for improved code autocomplete when the files were inside the `test` folder.